GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1551 @@
+            if (defaultDomain.Name != null)
+            {
+                Console.WriteLine("\nCreating a new user...");
+                Console.WriteLine("\n  Please enter first name for new user:");
+                String firstName = Console.ReadLine();
+                Console.WriteLine("\n  Please enter last name for new user:");
+                String lastName = Console.ReadLine();
+                newUser.DisplayName = firstName + " " + lastName;
+                newUser.UserPrincipalName = firstName + "." + lastName + Helper.GetRandomString(4) + "@" +
+                                            defaultDomain.Name;
+                newUser.AccountEnabled = true;
+                newUser.MailNickname = firstName + lastName;
+                newUser.PasswordProfile = new PasswordProfile
+                {
+                    Password = "ChangeMe123!",
+                    ForceChangePasswordNextLogin = true
+                };
+                newUser.UsageLocation = "US";
+                try
+                {
+                    await client.Users.AddUserAsync(newUser);
+                    Console.WriteLine("\nNew User {0} was created", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError creating new user {0}", Program.ExtractErrorMessage(e));
+                }
+            }
+            return newUser;
+
+            #endregion
+        }
+
+        private static async Task UpdateNewUser(IUser newUser)
+        {
+            #region Update User properties
+
+            //*******************************************************************************************
+            // update the newly created user's Password, PasswordPolicies and City
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                // update User's info
+                newUser.City = "Seattle";
+                newUser.Country = "UK";
+                newUser.Mobile = "+4477889456789";
+                newUser.UserType = "Member";
+
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} was updated:", newUser.DisplayName);
+                    Console.WriteLine("\t{0}:\t{1}", "City", "Seattle");
+                    Console.WriteLine("\t{0}:\t{1}", "Country", "UK");
+                    Console.WriteLine("\t{0}:\t{1}", "Mobile", "+4477889456789");
+                    Console.WriteLine("\t{0}:\t{1}", "Type", "Member");
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0}", Program.ExtractErrorMessage(e));
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task ResetUserPassword(IUser newUser)

```

Comment:
done

---
In reply to: [81251050](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r81251050) [](ancestors = 81251050)